code/_onclick/hud/alert.dm

```diff
@@ -430,7 +430,7 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 
 	var/mob/living/living_owner = owner
 	var/last_whisper = tgui_input_text(usr, "Do you have any last words?", "Final Words")
-	if (isnull(last_whisper) || !CAN_SUCCUMB(living_owner))
+	if (isnull(last_whisper) || !CAN_SUCCUMB(living_owner) || last_whisper == "")
```

Comment:
```suggestion
	if (!last_whisper || !CAN_SUCCUMB(living_owner))
```
The ! operator encompasses null, "" and 0.

Reply:
Today I learned an empty sting is untrue :)